resources/zib/zib-TNMTumorClassification.xml

```diff
@@ -0,0 +1,617 @@
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="TNMTumorClassification" />
+      <alias value="TNMTumorClassificatie" />
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.1" />
+        <comment value="TNMTumorClassification" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="385379008" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="DateOfTNMClassification" />
+      <definition value="The date at which the TNM Classification (the value) has been assigned." />
+      <alias value="DatumTNMClassificatie" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.2" />
+        <comment value="DateOfTNMClassification" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <max value="1" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Explanation" />
+      <definition value="Remark regarding interpretation of the integrated TNM value as is classified." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.5" />
+        <comment value="Explanation" />
+      </mapping>
+    </element>
+    <element id="Observation.method">
+      <path value="Observation.method" />
+      <short value="TNMVersion" />
+      <definition value="TNM classification version (AJCC or UICC) and edition used (7th, 8th etc.)." />
+      <alias value="TNMVersie" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.27.6--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.3" />
+        <comment value="TNMVersion" />
+      </mapping>
+    </element>
+    <element id="Observation.hasMember">
```

Comment:
I don't really understand why the R classification has been mapped to `.hasMember` instead of a `.component`. Do we know why this is done in https://simplifier.net/packages/de.basisprofil.onkologie/1.0.0-ballot/files/657938?

Reply:
If we however choose to keep the R classification as a `.hasMember`, then we should create a separate slice (copy from zib-LaboratoryTestResult).